bq/src/components/Login/Login.scss

```
@@ -0,0 +1,178 @@
+@import "../../styles/variables";
+.container__Login {
```

```suggestion
.container-Login {
```
La convención es suar __ para base de datos, para css con tranquilidad pueden usar -